### ISSUE TITLE
Errors with LDP-NR resources

### ISSUE THREAD

**Issue by acoburn** [scossu/lakesuperior#47]
### Environment

Operating system: OS X

Python version: 3.6

LAKEsuperior release, branch, or commit #: 1.0.0-alpha12 from pypi

### Steps to reproduce

Given a file in `file.txt` with the content:

```
this is a file.
```

And this request:

```sh
curl localhost:8000/ldp/ -XPOST -H"Content-Type: text/plain" -i --data-binary @file.txt
```

Then issue a GET request on the newly created resource.

```sh
curl -i <the resource>
```

### Observed behavior

```http
HTTP/1.1 500 INTERNAL SERVER ERROR
Server: gunicorn/19.7.1
Date: Mon, 09 Apr 2018 19:28:41 GMT
Connection: keep-alive
Content-Type: text/html
Content-Length: 291
```

### Expected behavior

The file contents. Or at least logs describing the error (there was nothing of interest in the error logs or in the console of the running application).

### Other notes worth mentioning

Requests to the metadata (e.g. <resource path>/fcr:metadata) respond correctly with RDF.

It is also worth noting that the binary is present on the filesystem in `./data/ldpnr_store`

**Comment by scossu** [comment]
@acoburn I cannot reproduce this in the master or development branch. 

What is the output in `./data/log/gunicorn-error.log`?

**Comment by acoburn** [comment]
For me there was _no_ output in the error log. I'll try it again tomorrow with a clean environment.

**Comment by scossu** [comment]
@acoburn Do you mind re-testing this with alpha 14?

**Comment by acoburn** [comment]
Tried with alpha 14:

```http
$ curl -i localhost:8000/ldp -XPOST -H"Content-Type: text/plain" --data-binary @file.txt
HTTP/1.1 500 INTERNAL SERVER ERROR
Server: gunicorn/19.7.1
Date: Wed, 25 Apr 2018 17:30:53 GMT
Connection: keep-alive
Content-Type: text/html
Content-Length: 291
```

Now I appear to be unable to POST the binary in the first place. (PUT continues to work fine)

**Comment by scossu** [comment]
Fixed in d0a76c0
